src/libs/codemirror-mgl.ts

```diff
@@ -41,19 +28,10 @@ CodeMirror.registerHelper("lint", "json", (text: string) => {
 
 CodeMirror.registerHelper("lint", "mgl", (text: string, opts: any, doc: any) => {
   const found: MarkerRangeWithMessage[] = [];
-  const {parser} = jsonlint as any;
   const {context} = opts;
 
-  parser.parseError = (str: string, hash: any) => {
-    const loc = hash.loc;
-    found.push({
-      from: CodeMirror.Pos(loc.first_line - 1, loc.first_column),
-      to: CodeMirror.Pos(loc.last_line - 1, loc.last_column),
-      message: str
-    });
-  };
   try {
-    parser.parse(text);
+    parse(text);
```

Comment:
Isn't the error handing logic is missing? 
Probably worth adding a test to make sure this is working as expected before and after this change.